### DISCUSSION TITLE
github not creating static page

### DISCUSSION THREAD

**Discussion by menv25** [community/community#48902] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I tried to create static for some projects using github pages but only one is working, the rest keep saying that the page does not exist, even the process of creating the page, according to github, having been done successfully,  have any criteria for creation?

**Comment by doguedogue** [comment, marked as answer]
It works fine for me, use an index.html page to start your projects

Take a look of my configuration
![image](https://user-images.githubusercontent.com/23409026/222201176-002756f7-d3e0-4532-a698-4e70cc870a5f.png)

You can also try another places to publish your projects like azure or heroku.

**Comment by menv25** [comment]
thank you, that was exactly it, the index.html was missing
